index.qmd

```diff
@@ -13,6 +13,10 @@ slug: index
 
 This Cookbook is learning-oriented to support scientific researchers who use NASA Earthdata as NASA migrates data and workflows to the cloud. It has resources to learn how to work with NASA Earthdata in the cloud, as well as documentation for how to contribute to these resources. It is under active, open development.
 
+**Tip: use the _earthaccess_ python library to access NASA earth data**. [earthaccess](https://earthaccess.readthedocs.io/en/latest/) is a Python library to access NASA earthdata locally or in the cloud. To learn more, see this [tech spotlight](), the [NASA Earthdata blog post](https://www.earthdata.nasa.gov/learn/blog/earthaccess), and these [how-to's](how-tos/find-data/find-python.html) and [tutorials](tutorials/). 
```

Comment:
```suggestion
**Tip: use the _earthaccess_ Python library to access NASA Earthdata**. [earthaccess](https://earthaccess.readthedocs.io/en/latest/) is a Python library that streamlines access to NASA Earthdata locally or in the cloud. To learn more, see <!--- this [tech spotlight](),---> the [NASA Earthdata blog post](https://www.earthdata.nasa.gov/learn/blog/earthaccess), and these [how-to's](how-tos/find-data/find-python.html) and [tutorials](tutorials/). 
```

Reply:
@ateucher I think it's best to merge this now, commenting out the tech spotlight till published, since coworking not till Thursday.

Reply:
@stefaniebutland link to tech spotlight: https://nasa-openscapes.github.io/news/2024-03-04-earthaccess-tech-spotlight/

Reply:
ah I get it! Thanks @jules32. I updated my suggested edit above. Ready to merge @ateucher